BashCompletion/README.md

```diff
@@ -0,0 +1,46 @@
+# Bash Completion
```

Comment:
This looks great!

It would be even better if there was a brief description of the "plugin" here, so when you read the page you immediately know why you would want to install it.

Reply:
@ahmadnurus I'd still like this to be added. What I'm looking for is just a one, maybe two, sentences to describe what this is and why it's useful.

Reply:
Sorry for my delay, I have added a brief description this bash completion